### PR TITLE
Alcohol Sections

### PR DESCRIPTION
**Pull request by floresj**
#914, #1190, #915, #916, #917, #918, #919, #920

### CODE REVIEW

**Comment by bmallred** [comment]
That's a numberwang!

**Comment by bmallred** [comment]
In the section under **Alcohol** and **Additional instances** the last question *"Did you successfully complete your counseling or treatment?"* there is a couple of questions:

 1. The wireframe states to show the explanation box if "Yes". Current behavior is on "No" and the content guide seems to suggest to show the explanation on either answer.
 1. The content guide has the text for the explanation box being *"Explanation for Yes or No"* and the wireframe has *"Did you successfully complete your counseling or treatment?"*. Which is preferred?

## Content guide

alcohol

## Wireframe

alcohol2

cc: @Jay-Neighbours-I72

**Comment by Jay-Neighbours-I72** [comment]
Let's load the explanation with a yes or no answer.

I missed that in the wireframe

**Comment by floresj** [comment]
Spec
substance-alcohol

File: src/config/locales/en.js [hunk 1]
```diff
@@ -6441,7 +6441,7 @@ const en = {
           completedTreatment: 'Did you successfully complete your counseling or treatment?',
           treatmentBeganDate: 'Provide the date counseling or treatment began',
           treatmentEndDate: 'Provide the date counseling or treatment ended',
-          noCompletedTreatment: 'You responded “No” to having successfully completed the treatment program. Provide explanation.'
+          noCompletedTreatment: 'Provide explanation'
```
Comment (bmallred) [inline src/config/locales/en.js]: Nice catch

**Pull request merged**